### ISSUE TITLE
Installation Error

### ISSUE THREAD

**Issue by talbaumel** [pythonnet/pythonnet#839]
### Environment

-   Pythonnet version: latest
-   Python version: 3.7
-   Operating System: Windows 10

### Details

-   While installing pythonnet `pip install pythonnet` I'm getting a .net error:

```
MSBuild auto-detection: using msbuild version '15.9.21.664' from 'C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\MSB
uild\15.0\bin'.
    C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\MSBuild\15.0\bin\Microsoft.CSharp.CurrentVersion.targets(322,5): erro
r MSB4019: The imported project "C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\MSBuild\15.0\bin\Roslyn\Microsoft.CSharp
.Core.targets" was not found. Confirm that the path in the <Import> declaration is correct, and that the file exists on disk. [C:\Users\
tabaumel\AppData\Local\Temp\pip-install-3voz84iv\pythonnet\src\runtime\Python.Runtime.csproj]
    C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\MSBuild\15.0\bin\Microsoft.CSharp.CurrentVersion.targets(322,5): erro
r MSB4019: The imported project "C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\MSBuild\15.0\bin\Roslyn\Microsoft.CSharp
.Core.targets" was not found. Confirm that the path in the <Import> declaration is correct, and that the file exists on disk. [C:\Users\
tabaumel\AppData\Local\Temp\pip-install-3voz84iv\pythonnet\src\testing\Python.Test.csproj]
    C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\MSBuild\15.0\bin\Microsoft.CSharp.CurrentVersion.targets(322,5): erro
r MSB4019: The imported project "C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\MSBuild\15.0\bin\Roslyn\Microsoft.CSharp
.Core.targets" was not found. Confirm that the path in the <Import> declaration is correct, and that the file exists on disk. [C:\Users\
tabaumel\AppData\Local\Temp\pip-install-3voz84iv\pythonnet\src\embed_tests\Python.EmbeddingTest.csproj]
    C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\MSBuild\15.0\bin\Microsoft.CSharp.CurrentVersion.targets(322,5): erro
r MSB4019: The imported project "C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\MSBuild\15.0\bin\Roslyn\Microsoft.CSharp
.Core.targets" was not found. Confirm that the path in the <Import> declaration is correct, and that the file exists on disk. [C:\Users\
tabaumel\AppData\Local\Temp\pip-install-3voz84iv\pythonnet\src\console\Console.csproj]
    C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\MSBuild\15.0\bin\Microsoft.CSharp.CurrentVersion.targets(322,5): erro
r MSB4019: The imported project "C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\MSBuild\15.0\bin\Roslyn\Microsoft.CSharp
.Core.targets" was not found. Confirm that the path in the <Import> declaration is correct, and that the file exists on disk. [C:\Users\
tabaumel\AppData\Local\Temp\pip-install-3voz84iv\pythonnet\src\clrmodule\clrmodule.csproj]
```

I'm new to .net and have no idea how to approach the error.
Thanks for support, Tal

**Comment by talbaumel** [comment]
After manually adding python to env path I got:

```
 Traceback (most recent call last):
    File "tools\geninterop\geninterop.py", line 292, in <module>
      sys.exit(main())
    File "tools\geninterop\geninterop.py", line 272, in main
      python_h = preprocess_python_headers()
    File "tools\geninterop\geninterop.py", line 192, in preprocess_python_headers
      for line in _check_output(cmd).splitlines():
    File "tools\geninterop\geninterop.py", line 41, in _check_output
      output = subprocess.check_output(*args, **kwargs)
    File "C:\Users\tabaumel\AppData\Local\Continuum\anaconda3\lib\subprocess.py", line 389, in check_output
      **kwargs).stdout
    File "C:\Users\tabaumel\AppData\Local\Continuum\anaconda3\lib\subprocess.py", line 466, in run
      with Popen(*popenargs, **kwargs) as process:
    File "C:\Users\tabaumel\AppData\Local\Continuum\anaconda3\lib\subprocess.py", line 769, in __init__
      restore_signals, start_new_session)
    File "C:\Users\tabaumel\AppData\Local\Continuum\anaconda3\lib\subprocess.py", line 1172, in _execute_child
      startupinfo)
  FileNotFoundError: [WinError 2] The system cannot find the file specified
  Traceback (most recent call last):
    File "<string>", line 1, in <module>
    File "C:\Users\tabaumel\AppData\Local\Temp\pip-install-_tgwnsmq\pythonnet\setup.py", line 405, in <module>
      zip_safe=False,
    File "C:\Users\tabaumel\AppData\Local\Continuum\anaconda3\lib\site-packages\setuptools\__init__.py", line 143, in setup
      return distutils.core.setup(**attrs)
    File "C:\Users\tabaumel\AppData\Local\Continuum\anaconda3\lib\distutils\core.py", line 148, in setup
      dist.run_commands()
    File "C:\Users\tabaumel\AppData\Local\Continuum\anaconda3\lib\distutils\dist.py", line 966, in run_commands
      self.run_command(cmd)
    File "C:\Users\tabaumel\AppData\Local\Continuum\anaconda3\lib\distutils\dist.py", line 985, in run_command
      cmd_obj.run()
    File "C:\Users\tabaumel\AppData\Local\Continuum\anaconda3\lib\site-packages\wheel\bdist_wheel.py", line 188, in run
      self.run_command('build')
    File "C:\Users\tabaumel\AppData\Local\Continuum\anaconda3\lib\distutils\cmd.py", line 313, in run_command
      self.distribution.run_command(command)
    File "C:\Users\tabaumel\AppData\Local\Continuum\anaconda3\lib\distutils\dist.py", line 985, in run_command
      cmd_obj.run()
    File "C:\Users\tabaumel\AppData\Local\Continuum\anaconda3\lib\distutils\command\build.py", line 135, in run
      self.run_command(cmd_name)
    File "C:\Users\tabaumel\AppData\Local\Continuum\anaconda3\lib\distutils\cmd.py", line 313, in run_command
      self.distribution.run_command(command)
    File "C:\Users\tabaumel\AppData\Local\Continuum\anaconda3\lib\distutils\dist.py", line 985, in run_command
      cmd_obj.run()
    File "C:\Users\tabaumel\AppData\Local\Continuum\anaconda3\lib\distutils\command\build_ext.py", line 339, in run
      self.build_extensions()
    File "C:\Users\tabaumel\AppData\Local\Continuum\anaconda3\lib\distutils\command\build_ext.py", line 448, in build_extensions
      self._build_extensions_serial()
    File "C:\Users\tabaumel\AppData\Local\Continuum\anaconda3\lib\distutils\command\build_ext.py", line 473, in _build_extensions_serial
      self.build_extension(ext)
    File "C:\Users\tabaumel\AppData\Local\Temp\pip-install-_tgwnsmq\pythonnet\setup.py", line 191, in build_extension
      subprocess.check_call([sys.executable, geninterop, interop_file])
    File "C:\Users\tabaumel\AppData\Local\Continuum\anaconda3\lib\subprocess.py", line 341, in check_call
      raise CalledProcessError(retcode, cmd)
  subprocess.CalledProcessError: Command '['C:\\Users\\tabaumel\\AppData\\Local\\Continuum\\anaconda3\\python.exe', 'tools\\geninterop\\geninterop.py', 'src\\runtime\\interop37.cs']' returned non-zero exit status 1.
```

**Comment by talbaumel** [comment]
Solved when switched to python 3.6.2

**Comment by filmor** [comment]
Yeah, I'm really sorry for this, I haven't yet gotten around releasing 2.4.0 which includes Python 3.7 support, I'll try it again this weekend.
